data/blog/governance.md

```diff
@@ -0,0 +1,38 @@
+
+---
+
+## Why are things changing?
+
+People in the Electron project coordinate in timezones around the world with volunteers, with full-time maintainers, and with several companies who all rely on Electron. Until now, we've been successful with informal coordination; but as the team has grown, we've found that the approach doesn't scale. We also want to make it easier for new contributors to find a place to call home in the project.
+
+## Working Groups
+
+Electron governance includes working groups that are responsible for different parts of the project. We're starting out with seven groups:
+ * Community & Safety: Handles [Code of Conduct](https://github.com/electron/governance/blob/master/CODE_OF_CONDUCT.md) issues.
+ * Docs & Tooling: Oversees externally-focused tooling (e.g. [Fiddle](https://electronjs.org/fiddle), [Forge](https://electronforge.io/)) and the Electron [documentation](https://electronjs.org/docs).
+ * Outreach: Helps grow the Electron community.
+ * Releases: Ensures releases are stable and on schedule.
+ * Security: Performs security testing and responds to security issues.
+ * Upgrades: Integrates upstream upgrades, such as new versions of V8, Chromium, and Node.
+ * Website: You're lookin' at it. https://electronjs.org/
+
+These groups will coordinate with each other, but each has their own meeting schedules and agendas to be productive on their own. More details on these groups is available [here](https://github.com/electron/governance/blob/master/README.md).
+
+## Does this change the Electron project's direction?
+
+This shouldn't have any direct effect on Electron's direction. With luck, working groups may make it easier for new contributors to find topics that interest them, and make maintainers' lives simpler by moving discussion they don't care about to other groups. If that happens, it may indirectly affect things by having more unblocked people working together.
+
+## Where can I learn more?
+
+ * The governance [repo](https://github.com/electron/governance/) and [charter](https://github.com/electron/governance/tree/master/charter) have information about the new governance structure.
+ * Each working group has its own page: [Community](https://github.com/electron/governance/tree/master/wg-community-safety), [Docs & Tools](https://github.com/electron/governance/tree/master/wg-docs-tools), [Outreach](https://github.com/electron/governance/tree/master/wg-outreach), [Releases](https://github.com/electron/governance/tree/master/wg-releases), [Security](https://github.com/electron/governance/tree/master/wg-security), [Upgrades](https://github.com/electron/governance/tree/master/wg-upgrades), and [Website](https://github.com/electron/governance/tree/master/wg-website).
+ * You can contact the maintainers by [opening an issue](https://github.com/electron/governance) or mailing us at [info@electronjs.org](mailto:info@electronjs.org).
```

Comment:
> * You can contact the maintainers by [opening an issue](https://github.com/electron/governance)

Should this link to https://github.com/electron/governance/issues instead? I'm not super picky on this.

Reply:
Yes, it should. I meant to leave it linking to /issues the way you had it; must've been a cut & paste error.

Reply:
Fixed @ ba8e017710f395172c5960ccc5c5ece6de232217